astropy/table/table.py

```diff
@@ -16,6 +16,7 @@
 
 from .. import log
 from ..io import registry as io_registry
+from ..io.fits.util import  _is_int
```

Comment:
So you want me to write a function `isint` as you have described above in `utils.misc`?
@taldcroft Any opinion?